addons/im_livechat/static/tests/sidebar_patch.test.js

```diff
@@ -407,8 +407,8 @@ test("unknown livechat can be displayed and interacted with", async () => {
     await contains(".o-mail-DiscussSidebarChannel.o-active", { text: "Jane" });
     await insertText(".o-mail-Composer-input", "Hello", { replace: true });
     await click(".o-mail-Composer-send:enabled");
-    await contains(".o-mail-Message", { text: "Hello" });
     await waitNotifications([env, "discuss.channel/new_message"]);
+    await contains(".o-mail-Message", { text: "Hello" });
```

Comment:
Why this should be changed? Order between the 2 await should be irrelevant here.